src/lib/components/particles/map-locator/index.jsx

```diff
@@ -0,0 +1,33 @@
+import defaultStyles from "./style.module.css"
+import { mergeStyles } from "$styles/helpers/mergeStyles"
+
+export const MapLocator = ({ path, projection, features, markerCoordinates, width, height, markerRadius=15, styles }) => {
+  styles = mergeStyles({ ...defaultStyles }, styles)
+
+  return (
+    <svg
+      class={styles.svg}
+      viewBox={`0 0 ${width} ${height}`}
+      preserveAspectRatio={"xMinYMin"}>
+      <g>
+        {
+          features.map((f, i) =>
+            <path
+              class={styles.path}
+              key={i}
+              d={path(f)}
```

Comment:
I'm worried that drawing these features on the frontend will end up being really slow, especially when we have any number of these little locators on the page. Could we extract the UK path into an SVG that we then load as a static asset, and perhaps provide this as a prop to the component?

As an aside, I wonder if this couldn't be part of a wider Map component, but that's something I'll look into at a later point.